File: ParadigmasdaProgramacao/atividadeCsharp/atividadeCsharp/Pessoa.cs
using System;

namespace atividadeCsharp
{
    public class Pessoa
    {
        public Pessoa()
        {
        }

        private string nome;
        private int idade;

       public static void pessoaMaisVelha(Pessoa pessoaUm, Pessoa pessoaDois){
            if (pessoaUm.GetIdade() > pessoaDois.GetIdade())
            {
                Console.WriteLine("Pessoa mais velha: " + pessoaUm.GetNome());
            }
            else {
                Console.WriteLine("Pessoa mais velha: " + pessoaDois.GetNome());

            }
        }
        public string GetNome()
        {
            return this.nome;
        }

        public void SetNome(string nome)
        {
            this.nome = nome;
        }

        public int GetIdade()
        {
            return idade;
        }

        public void SetIdade(int idade)
        {
            this.idade = idade;
        }
    }
}

File: ParadigmasdaProgramacao/atividadeJava/src/Questoes/Pessoa.java
package Questoes;

public class Pessoa {

    public Pessoa()
    {
    }

    private String nome;
    private int idade;

    public static void pessoaMaisVelha(Pessoa pessoaUm, Pessoa pessoaDois){
        if (pessoaUm.getIdade() > pessoaDois.getIdade())
        {
            System.out.println("Pessoa mais velha: " + pessoaUm.getNome());
        }
        else {
            System.out.println("Pessoa mais velha: " + pessoaDois.getNome());
        }
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }
}
